drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/CollectionViewImpl.java

```diff
@@ -174,6 +251,11 @@ public void setValue(String jsonString) {
         presenter.setValue(jsonString);
     }
 
+    @Override
+    public boolean isExpressionWidget() {
```

Comment:
@yesamer 
Since three possible paths (list/map/expression) I think it would be cleaner to define (another :) ) enum and use it

Reply:
@gitgabrio In this case I'm not agree, the expression property is no related to the Collection type. 
Here, I need to know if the user enable the "expression" collection editor, disregarding the Collection type.

Reply:
@gitgabrio I create https://issues.redhat.com/browse/DROOLS-4941 to manage this refactor during next sprint